### PR TITLE
ISPN-15080 Fix name-as-tags default in XSD

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-15080

### CODE REVIEW

File: core/src/main/resources/schema/infinispan-config-15.0.xsd [hunk 1]
```diff
@@ -809,7 +809,7 @@
         </xs:documentation>
       </xs:annotation>
     </xs:attribute>
-    <xs:attribute name="names-as-tags" type="xs:boolean" default="${GlobalMetrics.namesAsTags}">
```
Comment (fax4ever) [inline core/src/main/resources/schema/infinispan-config-15.0.xsd:812]: IIRC we have different other cases of camel-cased default

Comment (wburns) [reply core/src/main/resources/schema/infinispan-config-15.0.xsd:812]: I must admit it is also a bit confusing that the variable name is all upper case with underscores and the replaced attribute name is all lower case with dashes.

Comment (tristantarrant) [reply core/src/main/resources/schema/infinispan-config-15.0.xsd:812]: probably, but it is derived from the attribute name which is an enum which is converted from snake-uppercase to kebab-lowercase

**Comment by wburns** [comment]
Integrated into main, thanks @tristantarrant !

**Pull request merged**